fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/builders/ConjunctionDruidFilterBuilder.java

```diff
@@ -233,4 +234,16 @@ protected abstract Filter buildDimensionFilter(Dimension dimension, Set<ApiFilte
                 .map(finalFilterBuilder::apply)
                 .collect(Collectors.toList());
     }
+
+    /**
+     * Given a dimension and its contains filter values, return a list of Contains SearchFilters.
+     * @param dimension the dimension to search for
+     * @param values the contains value
+     * @return the list of SearchFilters
+     */
+    protected List<Filter> buildContainsSearchFilters(Dimension dimension, List<String> values) {
+        return values.stream()
+                .map(value -> new SearchFilter(dimension, SearchFilter.QueryType.Contains, value))
```

Comment:
should the query type be insensitive contains by default?

Reply:
we tested this, looks like this defaults to a case insensitive search for druid, but a case sensitive search for sql. May want to standardize this to be case insensitive for both.

Reply:
> 
> 
> @colemanProjects @kevinhinterlong The sql part is already tested: https://github.com/yahoo/fili/blob/master/fili-sql/src/test/groovy/com/yahoo/bard/webservice/sql/evaluator/FilterEvaluatorSpec.groovy

@zmz0305 If we make the sql search case insensitive then we should add a test

Reply:
@kevinhinterlong we decide to create another ticket for the SQL case insensitive search, so it won't be in this PR